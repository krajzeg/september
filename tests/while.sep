from := 9; to := 3

print("Iterating over numbers, starting at:")
print(from)
print("And ending at:")
print(to)
print("======")

dir := if (from > to) {-1} else {1}
while (from != to) {
	print(from)
	from = from + dir	
}
print(from)

print("======")
print("While loop exited cleanly.")