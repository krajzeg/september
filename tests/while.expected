Iterating over numbers, starting at:
9
And ending at:
3
======
9
8
7
6
5
4
3
======
While loop exited cleanly.
